### PR TITLE
[OSDOCS-4998]: adding diagrams to hypershift DR doc

### PR DESCRIPTION
**Pull request by lahinson**
Version(s): 4.12+

Issue: https://issues.redhat.com/browse/OSDOCS-4998

Link to docs preview: https://55852--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/control_plane_backup_and_restore/disaster_recovery/dr-hosted-cluster-within-aws-region.html#dr-hosted-cluster-process

QE review:
- [x] QE has approved this change.

Additional information: This PR adds diagrams and some additional text to the information in the hosted control planes DR docs.

### CODE REVIEW

**Comment by lahinson** [comment]
FYI @jparrill @jiezhao16 - PTAL when you can. Thank you!

**Comment by lahinson** [comment]
@jparrill Do you have some time this week to review these doc updates?

File: backup_and_restore/control_plane_backup_and_restore/disaster_recovery/dr-hosted-cluster-within-aws-region.adoc [hunk 1]
```diff
@@ -104,17 +104,25 @@ EXTERNAL_DNS_DOMAIN="hc.jpdv.aws.kerbeross.com"
...
 
-. On management cluster 1, which you can think of as the source management cluster, the control plane and workers interact by using the ExternalDNS API.
+. On management cluster 1, which you can think of as the source management cluster, the control plane and workers interact by using the external DNS API. The external DNS API is accessible, and a load balancer sits between the management clusters.
++
+image::298_OpenShift_Backup_Restore_0123_00.png[Diagram that shows the workers accessing the external DNS API and the external DNS API pointing to the control plane through a load balancer]
 
-. You take a snapshot of the hosted cluster, which includes etcd, the control plane, and the worker nodes. The worker nodes are moved to the external DNS, the control plane is saved in a local manifest file, and etcd is backed up to an S3 bucket.
+. You take a snapshot of the hosted cluster, which includes etcd, the control plane, and the worker nodes. The worker nodes are moved to the external DNS API, the control plane is saved in a local manifest file, and etcd is backed up to an S3 bucket. The data plane is active and the control plane is paused.
```
Comment (jparrill) [inline backup_and_restore/control_plane_backup_and_restore/disaster_recovery/dr-hosted-cluster-within-aws-region.adoc:111]: s/The worker nodes are moved to the external DNS API/The worker nodes still try to access to the External DNS API even if it's not accesible, but the workloads will be still running./

**Review by jparrill — CHANGES_REQUESTED** [review]
dropped some comments :)

**Review by jparrill — COMMENTED** [review]
/lgtm

File: backup_and_restore/control_plane_backup_and_restore/disaster_recovery/dr-hosted-cluster-within-aws-region.adoc [hunk 2]
```diff
@@ -104,17 +104,25 @@ EXTERNAL_DNS_DOMAIN="hc.jpdv.aws.kerbeross.com"
...
+. On management cluster 1, which you can think of as the source management cluster, the control plane and workers interact by using the external DNS API. The external DNS API is accessible, and a load balancer sits between the management clusters.
++
+image::298_OpenShift_Backup_Restore_0123_00.png[Diagram that shows the workers accessing the external DNS API and the external DNS API pointing to the control plane through a load balancer]
 
-. You take a snapshot of the hosted cluster, which includes etcd, the control plane, and the worker nodes. The worker nodes are moved to the external DNS, the control plane is saved in a local manifest file, and etcd is backed up to an S3 bucket.
+. You take a snapshot of the hosted cluster, which includes etcd, the control plane, and the worker nodes. The worker nodes continue to try to access the external DNS API even if it is not accessible, the workloads are running, the control plane is saved in a local manifest file, and etcd is backed up to an S3 bucket. The data plane is active and the control plane is paused.
```
Comment (mburke5678) [inline backup_and_restore/control_plane_backup_and_restore/disaster_recovery/dr-hosted-cluster-within-aws-region.adoc:111]: The _The worker nodes continue to try to access_ sentence confuses me a bit. It sounds like the worker nodes will continue if the following 4 conditions exist. I don't think that is true, though. Would splitting the sentence after _it is not accessible_ and re-wording the rest slightly to make a second sentence be right?

Comment (mburke5678) [reply backup_and_restore/control_plane_backup_and_restore/disaster_recovery/dr-hosted-cluster-within-aws-region.adoc:111]: Oooh. I like the _During this process_ in the next step. Would that work here?

**Comment by mburke5678** [comment]
@lahinson One small nit. Otherwise LGTM

File: backup_and_restore/control_plane_backup_and_restore/disaster_recovery/dr-hosted-cluster-within-aws-region.adoc (hunk 2 above)
Comment (lahinson) [reply backup_and_restore/control_plane_backup_and_restore/disaster_recovery/dr-hosted-cluster-within-aws-region.adoc:111]: I'll try it!

Comment (lahinson) [reply backup_and_restore/control_plane_backup_and_restore/disaster_recovery/dr-hosted-cluster-within-aws-region.adoc]: @mburke5678 I incorporated your suggestion. Thanks!

**Pull request merged**